src/components/badges/BadgeSeverity.tsx

```diff
@@ -261,6 +263,11 @@ const StyledSeverityText = styled.span`
 `;
 StyledSeverityText.displayName = 'StyledSeverityText';
 
+const StyledSeverityIcon = styled.span`
+  color: var(--badge-severity-icon-color);
+`;
+StyledSeverityIcon.displayName = 'StyledSeverityIcon';
```

Comment:
I think you can just set it in StyledSeverityContent instead of creating a new component